pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -350,3 +350,75 @@ func CreateObjects(client *client.Client, objects []runtime.Object) {
 	}
 	fmt.Println("\nApplication has been deployed to Kubernetes. You can run 'kubectl get deployment,svc' for details.")
 }
+
+func DeleteObjects(client *client.Client, name string) {
+	err := client.Services(api.NamespaceDefault).Delete(name)
+	if err != nil {
+		logrus.Fatalf("Error: '%v' while deleting service: %s", err, name)
```

Comment:
Rather then failing after fist error, we should just display error, and continue to try delete next object.
Could we return this error, and handle it in `func Down(c *cli.Context)`?